lib/lf/geometry/test/geometry_tests.cc

```diff
@@ -0,0 +1,91 @@
+/**
+ * @file
+ * @brief tests for geometry objects
+ * @author Anian Ruoss
+ * @date   2018-10-27 15:57:17
+ * @copyright MIT License
+ */
+
+#include <gtest/gtest.h>
+#include <lf/geometry/geometry.h>
+
+using GeometryObjects =
+    testing::Types<lf::geometry::TriaO1, lf::geometry::QuadO1>;
+
+static const Eigen::Matrix2d map_ = [] {
+  Eigen::Matrix2d tmp;
+  tmp << 1, 2, 3, 4;
+  return tmp;
+}();
```

Comment:
tip: you can also use `static const Eigen::Matrix2d map_ = (Eigen::Matrix2d() << 1,2,3,4).finished()`

Reply:
I like it, thanks :+1: